docs/third-party-developers/extensibility/data-store/schema.md

```diff
@@ -81,6 +84,47 @@ This will return all the registered routes for the given namespace as a flat arr
 
 -   `array`: An array of all routes for the given namespace.
 
+### getRouteFromResourceEntries
+
+This will returns the route from the given slice of the route state.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   _stateSlice_ `object`: Slice of the route state from a given namespace and resource name.
+-   _ids_ `array`: An array of id references that are to be replaced in route placeholders. If the route does not have placeholders, this can be an empty array.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `string`: The route for the given resource entries, or `null` if no route is found.
```

Comment:
What are your thoughts on this format so that a developer quickly sees all return types:

```suggestion
-   `string`|`null`: The route for the given resource entries, or null if no route is found.
```